dawg_python/wrapper.py

```diff
@@ -17,29 +17,29 @@ def __init__(self):
     "Root index"
 
     def has_value(self, index):
-        "Checks if a given index is related to the end of a key."
+        #Checks if a given index is related to the end of a key.
```

Comment:
Fair. I changed it because PEP8 checks were complaining, and I never use docs at runtime, so forgot about that. I'm pretty neutral on this, so I'll change them back.